### ISSUE TITLE
Custom Physics (Enhancement)

### ISSUE THREAD

**Issue by ccdebest** [Ukendio/Game#21]
The game will need some unique features to Roblox. I haven't seen a game on Roblox with Titanfall movement before, and it would be a nice addition. We can start this by making an engine that will replicate Arsenal's wacky movement, and then add features that will be unique to our game. Features that I would like to see are wall runs, grabble, air strafing.

**Comment by Ukendio** [comment]
Yes, definitely think there can be some grappling mechanics and air strafing. I was thinking about pathfinder (apex legends) for reference! Does add a considerable amount of complexity, so will have to wait till the game matures.

**Comment by ccdebest** [comment]
Alright! I can make a model for pathfinders grapple if you want
